main/common/aperture.js

```diff
@@ -43,7 +43,7 @@ const startRecording = async options => {
     cropArea: cropperBounds,
     showCursor,
     highlightClicks,
-    displayId: String(displayId)
+    screenId: displayId
```

Comment:
This is for the Aperture upgrade. See: https://github.com/wulkano/aperture/commit/38f82228c02e1e357369768d739792f835b07944